app/uk/gov/hmrc/tai/model/domain/AnnualAccount.scala

```diff
@@ -31,7 +31,7 @@ case class AnnualAccount (
     if (payments.isEmpty) 0 else payments.max.amountYearToDate
 
   lazy val totalTaxPaidYearToDate: BigDecimal = {
-    payments.max.taxAmountYearToDate
+    if (payments.isEmpty) 0 else payments.max.taxAmountYearToDate
   }
```

Comment:
Slightly duplicated with the line above, you can pull out a common function here